app/controllers/bookable_slots_controller.rb

```diff
@@ -1,7 +1,30 @@
 class BookableSlotsController < ApplicationController
+  def index
+    slots = BookableSlot
+            .without_holidays
+            .within_date_range(start_at, end_at)
+    slots = slots.where(guider: current_user) if scoped_to_me?
+    json = ActiveModelSerializers::SerializableResource.new(slots).to_json
+    render json: json
```

Comment:
I think you can just `render json: slots` here and the serializer is correctly inferred...